src/vsr/superblock_quorums_fuzz.zig

```diff
@@ -122,11 +122,12 @@ fn test_quorums_working(
         header.* = std.mem.zeroInit(SuperBlockHeader, .{
             .copy = @intCast(u8, i),
             .version = SuperBlockVersion,
-            .replica = 1,
             .storage_size_max = superblock.data_file_size_min,
             .sequence = copies[i].sequence,
             .parent = checksums[copies[i].sequence - 1],
         });
+        header.vsr_state.replica = 1;
+        header.vsr_state.replica_count = 6;
```

Comment:
It's a bit annoying that there's so much different tests where we construct sb header manually...

Reply:
Yeah, superblock bridges the LSM & consensus, so it ends up all over.